cloudferry_devlab/cloudferry_devlab/tests/functional_test.py

```diff
@@ -23,6 +23,7 @@
 from cloudferry_devlab import generate_load
 from cloudferry_devlab.tests import test_exceptions
 import cloudferry_devlab.tests.utils as utils
+from cloudferry_devlab.tests.base import get_nosetest_cmd_attribute_val
```

Comment:
Do not import objects, only modules (*)
http://docs.openstack.org/developer/hacking/#imports